tests/copyright_check/copyright_validator.py

```diff
@@ -0,0 +1,148 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from __future__ import print_function
+
+import difflib
+
+import os
+import re
+import sys
+import pathlib
+import config
+
+print_logs = config.print_logs
```

Comment:
I'm intrigued by the fact that we use the builtin print method to maybe print the logs, and we achieve it by writing into a sink file descriptor.

I'm wondering if having something like

```python
def print_logs(*args, **kwargs):
    if not config.debug_logs:
        return
    print(*args, **kwargs)
```

would be more effective